app/javascript/apps/Projects/index.tsx

```diff
@@ -26,17 +42,24 @@ export const ProjectsIndex: React.FC<ProjectsProps> = ({
             Team: {params.tags[0]}
           </Sans>
         )}
-        {unreleasedProjects && (
-          <Box pb={4}>
-            <Sans size="6">Out of sync</Sans>
-            <ProjectSummaryGrid projects={unreleasedProjects} />
-          </Box>
-        )}
-        {releasedProjects && (
-          <Box pb={4}>
-            <Sans size="6">Up to date</Sans>
-            <ProjectSummaryGrid projects={releasedProjects} />
-          </Box>
+        {ProjectsViewSwitcher()}
+        {isListView ? (
```

Comment:
It would be kinda cool to see this look more like this:

```ts
{isListView ? <ProjectsList projects={projects} /> : <ProjectsGrid projects={projects} />}
```

Or something like that anyway - I guess what I'm saying is that with this switcher, I think we have two concepts that could be extracted and then here I'd want to see the same levels of abstraction used. What do you think?